PocketKit/Sources/Textile/Style/Paragraph/ParagraphStyle.swift

```diff
@@ -0,0 +1,11 @@
+public struct ParagraphStyle {
```

Comment:
This is a great addition early-on. It'll come in super-handy once we include support for line height adjustment n' such.